Cargo.toml

```diff
@@ -51,6 +51,7 @@ async-trait = "0.1.68"
 axum = { version = "0.6.18", features = ["macros", "multipart", "ws"] }
 base64 = "0.21.0"
 blake3 = "1.3.3"
+bollard = "0.15.0"
```

Comment:
Aren't we using a different library for the `si` binary? Feels like we should pick one and throw it in a theoretical `lib/si-docker` wrapper crate.
cc @stack72 @fnichol

Reply:
yes we use [docker-rs](https://github.com/vv9k/docker-api-rs)

Reply:
We talked very briefly about this the other day. The consensus was that `bollard` is a little more feature rich and supported, so if we're going to create a wrapper we should probably use that.